geonode/maps/templates/maps/map_include.html

```diff
@@ -31,6 +31,50 @@
         portalConfig: {
             height: 600,
             renderTo: "the_map"
+        },
+
+        listeners: {
+            "ready": function() {
+                app.mapPanel.map.getMaxExtent = function() {
+                    return new OpenLayers.Bounds(-80150033.36,-80150033.36,80150033.36,80150033.36);
```

Comment:
should those values be hardcoded here?

Reply:
Could be improved of course. I just keep the original version and applied the minimal set of changes to let it working as expected.

Reply:
thanks